portfolio/src/main/webapp/script.js

```diff
@@ -15,27 +15,34 @@
 /**
  * Adds a random activity to the page.
  */
-function addRandomGreeting() {
-  const greetings =
+function addRandomActivity() {
```

Comment:
Thanks for renaming your functions and variables; they are easier to read :) Bravo